.pyproject_generation/pyproject_custom.toml

```
@@ -0,0 +1,15 @@
+[project]
+name = "ifrs"
+version = "1.2.0"
+description = "Internal-File-Registry-Service - This service acts as a registry for the internal location and representation of files."
+dependencies = [
+    "ghga-event-schemas==1.0.0",
+    "ghga-service-commons>=2.0.0",
+    "hexkit[mongodb,s3,akafka]>=2.0.0",
```

`clear_topics` is used, so the lower bound needs to be adjusted:
```suggestion
    "hexkit[mongodb,s3,akafka]>=2.1.0",
```